modules/exploits/linux/http/apache_ofbiz_deserialization.rb

```diff
@@ -55,7 +55,7 @@ def initialize(info = {})
               'Platform' => 'linux',
               'Arch' => [ARCH_X86, ARCH_X64],
               'Type' => :linux_dropper,
-              'CmdStagerFlavor' => %i[curl wget],
+              'CmdStagerFlavor' => %i[bourne curl wget],
```

Comment:
I no longer set this array, so removing it is also fine. It's far too annoying.

Reply:
And `CMDSTAGER::FLAVOR` is already set. That's the one that matters, IMHO. You can change it to `:bourne` if you'd like.